plugins/automation/plugin.rb

```diff
@@ -181,6 +182,10 @@ def self.get_active_automation
     DiscourseAutomation::EventHandlers.handle_pm_created(topic) if topic.private_message?
   end
 
+  on(:topic_tags_changed) do |topic, old_tags, new_tags|
```

Comment:
I'm looking at the trigger code for this event:

https://github.com/discourse/discourse/blob/11369018b6841b7afa1a564858cbefb710b05be9/lib/discourse_tagging.rb#L210-L215

I think there are just 2 arguments, one for the topic and the other is a hash that contains the added/removed tags array?

Reply:
You were right! 

I have not used it more than 1 arg in my triggers, so I just assumed it was placing every argument as it is.

It folds into a hash, which makes things easier